core/src/main/scala/org/bykn/bosatsu/TypedExpr.scala

```diff
@@ -101,6 +101,37 @@ sealed abstract class TypedExpr[T] {
         (expr.traverseType(fn), tbranch).mapN(Match(_, _, tag))
     }
 
+  def traverseTag[F[_]: Applicative, S](fn: T => F[S]): F[TypedExpr[S]] =
```

Comment:
this is the normal `Traverse[TypedExpr]` instance, which I used to have, but I removed since I wasn't using it.

Can we instead make this:
```scala
implicit val traverseTypedExpr: Traverse[TypedExpr] =
  new Traverse....
```
we can actually use cats-laws to test that, but it is a bit of a pain since we have to generate random TypedExpr. Maybe easier to test it by calling it in your code.

Reply:
Totally!